db/schema.rb

```diff
@@ -260,7 +259,7 @@
     t.index ["wiki_id"], name: "index_courses_wikis_on_wiki_id"
   end
 
-  create_table "faqs", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci", force: :cascade do |t|
+  create_table "faqs", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
```

Comment:
I assume these schema.rb changes are unrelated noise.

Reply:
yes! This schema change always creeps up in my git diff everytime which is annoying. I'll revert this.